.prow.yaml

```diff
@@ -47,8 +47,35 @@ presubmits:
             - test
           env:
             - name: USE_GOTESTSUM
-              value: '1'
+              value: "1"
           resources:
             requests:
               memory: 4Gi
               cpu: 2
+
+# Simple Tide configuration to prevent premature merges
+tide:
+  queries:
+    - repos:
+        - kubestellar/kubestellar
+      labels:
+        - lgtm
+        - approved
+      missingLabels:
+        - do-not-merge
+        - do-not-merge/hold
+        - do-not-merge/work-in-progress
+        - needs-rebase
```

Comment:
Do we need this here? We already have https://github.com/kubestellar/infra/blob/main/prow/config.yaml#L102-L117 .